src/course-unit/course-xblock/xblock-messages/XBlockMessages.jsx

```diff
@@ -0,0 +1,46 @@
+import PropTypes from 'prop-types';
+import { Alert } from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import { Info as InfoIcon, WarningFilled as WarningIcon } from '@edx/paragon/icons';
+
+import messages from '../messages';
+import { MESSAGE_ERROR_TYPES } from '../constants';
+import { getMessagesBlockType } from './utils';
+
+const XBlockMessages = ({ validationMessages }) => {
+  const intl = useIntl();
+  const type = getMessagesBlockType(validationMessages);
+
+  if (!validationMessages.length) {
+    return null;
+  }
+
+  return (
+    <Alert
+      variant={type === MESSAGE_ERROR_TYPES.warning ? 'warning' : 'danger'}
```

Comment:
[question]: Can we put this code into a variable?
```suggestion
     const { warning } = MESSAGE_ERROR_TYPES;
     const alertVariant = (type === warning) ? 'warning' : 'danger';
     ...
     variant={alertVariant}
```